custom_code/hooks.py

```diff
@@ -9,7 +11,18 @@ def target_post_save(target, created):
     """This hook runs following update of a target."""
     logger.info('Target post save hook: %s created: %s', target, created)
 
-    qprob, qso, qoffset = static_cats_query([target.ra], [target.dec])
+    qprob, qso, qoffset, asassnprob, asassn, asassnoffset = static_cats_query([target.ra], [target.dec])
```

Comment:
Can you add the specific version of `kne_cand_vetting` to `requirements.txt`? Otherwise people will have problems trying to run the updated TOM in an old environment.